pkg/scheduler/factory_test.go

```diff
@@ -553,6 +554,53 @@ func TestDefaultErrorFunc_NodeNotFound(t *testing.T) {
 	}
 }
 
+func TestDefaultErrorFunc_PodAlreadyBound(t *testing.T) {
+	nodeFoo := &v1.Node{ObjectMeta: metav1.ObjectMeta{Name: "foo"}}
+	testPod := &v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "test-pod", Namespace: "default"}, Spec: v1.PodSpec{NodeName: "foo"}}
```

Comment:
not sure if that possible, what I want to see is the nodename is set after the timeout of bind operation, and this testcase doesn't expose the issue around the bind.